.github/workflows/ci.yml

```
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  push:
+    branches: [main, polaris-release]
+  pull_request:
+
+jobs:
+  build:
+    name: 'Build, Lint, and Validate with Node v${{ matrix.node_version }}'
```

```suggestion
    name: 'Validate with Node v${{ matrix.node_version }}'
```

To match the names in the shipit files

---

Good catch!